huxley/api/serializers/school.py

```diff
@@ -49,6 +50,7 @@ class Meta:
             'fees_owed',
             'fees_paid',
             'assignments_finalized',
+            'delegate_names_finalized',
```

Comment:
We should probably add some validation to the delegate update view to prevent it from being edited if the school's delegates are finalized. Currently, we have this boolean field but it doesn't do anything.